run_aprime.bash

```diff
@@ -353,35 +343,48 @@ if [ ! -d $www_dir/$plots_dir_name ]; then
 else
   rm -f $www_dir/$plots_dir_name/*
 fi
+# Check on MPAS-Analysis www_dir, permissions included
+# Create it if it does not exist, purge it if it does
```

Comment:
this below is a little convoluted, but it was necessary to change permissions in all subdirectories of `www_dir`, since `chmod -R` only works from the last subdirectory down. If somebody has a better solution please let me know.

Reply:
I don't think this is the stage where you want to do the chmod, as I mention below.  So I wouldn't worry about that issue here.

Reply:
you're right. I'll move it at the end.

Reply:
I think I understand this better now.  You have a separate folder `mpas-analyis` under `$www_dir`, with folders containing the MPAS analysis websites.  

Why not just have an `mpas-analysis` subdirectory inside of `$www_dir/$plots_dir_name/`?  Then, all you have to do is link to `mpas-analysis/index.html` (or its subdirectories).  You shouldn't need to make the directory or chmod it here.  You could delete it if you want to make sure you're starting fresh.